domain/application/state/unit.go

```diff
@@ -756,15 +762,7 @@ WHERE  unit_uuid = $unitUUID.uuid
 //     doesn't exist or;
 //   - error satisfying [applicationerrors.ApplicationIsDead] if the application
 //     is dead.
-func (st *State) GetUnitWorkloadStatusesForApplication(ctx context.Context, appID coreapplication.ID) (map[coreunit.Name]application.StatusInfo[application.WorkloadStatusType], error) {
-	type statusInfoAndUnitName struct {
-		UnitName  coreunit.Name `db:"name"`
-		StatusID  int           `db:"status_id"`
-		Message   string        `db:"message"`
-		Data      []byte        `db:"data"`
-		UpdatedAt *time.Time    `db:"updated_at"`
-	}
```

Comment:
@gfouillet won't be happy!

Reply:
I'll repeat for the sake of all - **all** types go in types.go.